Docs/developer_guide/modules/transforms.md

```diff
@@ -1,11 +1,11 @@
 # Transforms
 
 ## Related MRML nodes
-- [vtkMRMLTransformableNode](https://slicer.org/doc/html/classvtkMRMLTransformableNode.html): any node that can be transformed
-- [vtkMRMLTransformNode](https://slicer.org/doc/html/classvtkMRMLTransformNode.html): it can store any linear or deformable transform or composite of multiple transforms
-  - [vtkMRMLLinearTransformNode](https://slicer.org/doc/html/classvtkMRMLLinearTransformNode.html): Deprecated. The transform does exactly the same as vtkMRMLTransformNode but has a different class name, which are still used for showing only certain transform types in node selectors. In the future this class will be removed. A vtkMRMLLinearTransformNode may contain non-linear components after a non-linear transform is hardened on it. Therefore, to check linearity of a transform the vtkMRMLTransformNode::IsLinear() and vtkMRMLTransformNode::IsTransformToWorldLinear() and vtkMRMLTransformNode::IsTransformToNodeLinear() methods must be used instead of using vtkMRMLLinearTransformNode::SafeDownCast(transform)!=NULL.
-  - [vtkMRMLBSplineTransformNode](https://slicer.org/doc/html/classvtkMRMLBSplineTransformNode.html): Deprecated. The transform does exactly the same as vtkMRMLTransformNode but has a different class name, which are still used for showing only certain transform types in node selectors. In the future this class will be removed.
-  - [vtkMRMLGridTransformNode](https://slicer.org/doc/html/classvtkMRMLGridTransformNode.html): Deprecated. The transform does exactly the same as vtkMRMLTransformNode but has a different class name, which are still used for showing only certain transform types in node selectors. In the future this class will be removed.
+- [vtkMRMLTransformableNode](https://apidocs.slicer.org/main/classvtkMRMLTransformableNode.html): any node that can be transformed
```

Comment:
DOC files for stable releases should be hardened down to their relevant version? If I was reviewing the ReadTheDocs for v5.8, I would suspect it would like me to 5.8 API documentation.

```suggestion
- [vtkMRMLTransformableNode](https://apidocs.slicer.org/v5.8/classvtkMRMLTransformableNode.html): any node that can be transformed
```

Reply:
Or this gets handled by https://github.com/Slicer/Slicer/pull/8263 and this PR should be integrated first before that other backport?

Reply:
This can be integrated as is. The versioning will indeed be handled in https://github.com/Slicer/Slicer/pull/8263.